scripts/common/object_store.sh

```diff
@@ -150,6 +152,7 @@ function migrate_between_object_stores() {
         if kubernetes_resource_exists kurl deployment ekc-operator; then
             kubectl -n kurl scale deploy ekc-operator --replicas=1
         fi
+        kubectl logs -n "default" sync-object-store || true
```

Comment:
i dont see why this is necessary. it seems like logs on line 119 should be enough and this will only duplicate the logs

Reply:
I update the above line. We should only get the logs when it fails/timeout is faced.
See.

Reply:
As i see it this line will only execute when the other logs command doesnt if the pod did not start. I would prefer you to remove this line as it adds no value.